README.md

```diff
@@ -64,3 +64,38 @@ Zdroje jsou zveřejněny pod [licencí BSD 3-Clause](LICENSE).
 **Tech leads:** [Matěj 'Horm' Horák](https://github.com/HormCodes), [Míla Votradovec](https://github.com/miiila)
 
 **Wiki:** [Redesign webu Česko.Digital](https://wiki.cesko.digital/pages/viewpage.action?pageId=1574868)
+
+## Translations
+
+Překlady jsou realizovány pomocí `gatsby-plugin-react-i18next` ve spojení se Airtable. Airtable umožnuje vytvářet tabulky a databáze, ze kterých jsou překlady staženy pomocí skriptu `yarn translations:get`. Tento skript spustí `scripts/get-translations.ts`. V rámci tohoto skriptu je vytvořeno připojení do Airtable, staženy záznamy a vytvořeny JSON soubory ve složce `locale/`.
```

Comment:
Také by se hodila zmínka o `gatsby clean`

Reply:
Pridano a zaroven jsem to dal jako soucast `yarn translations:get`